.github/workflows/test-build.yaml

```
@@ -57,8 +57,8 @@ jobs:
           repository: 'smartsheet-sdk-tests'
       - name: Setup Mock API
         run: |
-          smartsheet-sdk-tests/ci_scripts/install_wiremock.sh
-          smartsheet-sdk-tests/ci_scripts/start_wiremock.sh
+          docker compose -f smartsheet-sdk-tests/docker-compose.yml up -d
+          sleep 5
```

nit:
```suggestion
healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:8080/<whatever is the endpoint>"]
      interval: 2s
      timeout: 5s
      retries: 5
      start_period: 5s
```

more code but is more docker friendly. Then we only do `docker compose -f smartsheet-sdk-tests/docker-compose.yml up -d --wait`

---

Good one, the wiremock image already has a health check this means we only need to add --wait to the compose comand.
@denislavstanchev can you prepare PRs for the rest of the SDKs?